Dockerfile

```diff
@@ -1,8 +1,8 @@
 FROM alpine:latest
-LABEL version="2.2" maintainer="JJMerelo@GMail.com" perl6version="2019.11"
+LABEL version="2.3" maintainer="JJMerelo@GMail.com" perl6version="2019.11"
 
 # Environment
-ENV PATH="/root/.rakudobrew/versions/moar-2019.11/install/bin:/root/.rakudobrew/versions/moar-2019.11/install/share/perl6/site/bin:/root/.rakudobrew/bin:${PATH}" \
+ENV PATH="/root/raku-install/bin:/root/raku-install/share/perl6/site/bin:/root/.rakudobrew/bin:${PATH}" \
     PKGS="curl git perl" \
```

Comment:
Perl is only used in rakudobrew. It's not needed if rakudobrew is eliminated.

Reply:
I decided to leave the perl because didn't know your plans for it. Now I removed it :)
101M - one-fifth of the previous size.